testing/trino-product-tests/src/main/java/io/trino/tests/product/phoenix/TestPhoenix.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.phoenix;
+
+import io.trino.tempto.ProductTest;
+import io.trino.tempto.query.QueryResult;
+import org.testng.annotations.Test;
+
+import static io.trino.tempto.assertions.QueryAssert.Row.row;
+import static io.trino.tempto.assertions.QueryAssert.assertThat;
+import static io.trino.tempto.query.QueryExecutor.query;
+import static io.trino.tests.product.TestGroups.PHOENIX;
+import static io.trino.tests.product.TestGroups.PROFILE_SPECIFIC_TESTS;
+import static io.trino.tests.product.utils.QueryExecutors.onTrino;
+import static java.lang.String.format;
+
+public class TestPhoenix
+        extends ProductTest
+{
+    @Test(groups = {PHOENIX, PROFILE_SPECIFIC_TESTS})
+    public void testCreateTableAsSelect()
+    {
+        String tableName = "phoenix.default.nation";
+        QueryResult result = onTrino().executeQuery(format("CREATE TABLE %s AS SELECT * FROM tpch.tiny.nation", tableName));
```

Comment:
Can we use some generic set of tests? Where you could set schema and catalog in `tempto-configration.yaml`.

Reply:
I have partially applied this comment.